rued-esp32/src/peripherals.rs

```diff
@@ -31,10 +32,12 @@ pub struct SystemPeripherals<
     CHAN2PIN,
 > {
     pub pulse_counter: PulseCounterPeripherals<P>,
-    pub valve: ValvePeripherals,
+    // pub valve: ValvePeripherals,
```

Comment:
Disabled this as I'm running out of valid GPIO.  I can only use the available pins here https://cdn.sparkfun.com/assets/learn_tutorials/1/2/0/8/MicroModESP32ProcessorBoard_a.pdf